xla/service/gpu/transforms/windowed_einsum_handler.cc

```diff
@@ -1163,19 +1154,20 @@ absl::StatusOr<bool> WindowedEinsumHandler::Run(
       5, "WindowedEinsumHandler::Run(), before:\n" + module->ToString());
   bool changed = false;
   int64_t stream_id = hlo_query::NextChannelId(*module);
-
+  std::vector<HloInstruction*> all_windowed_einsum_loops;
   for (HloComputation* comp :
        module->MakeNonfusionComputations(execution_threads)) {
     if (comp->name().find(kWindowedEinsumRsLoopName) == 0) {
       VLOG(5) << "Processing computation: " << comp->name();
-      TF_ASSIGN_OR_RETURN(bool comp_result,
-                          HandleRsWindowedEinsumLoop(comp, stream_id));
+      TF_ASSIGN_OR_RETURN(bool comp_result, HandleRsWindowedEinsumLoop(comp));
       changed = comp_result;
+      all_windowed_einsum_loops.push_back(comp->WhileCallInstruction());
     } else if (comp->name().find(kWindowedEinsumAgLoopName) == 0) {
       VLOG(5) << "Processing computation: " << comp->name();
-      TF_ASSIGN_OR_RETURN(changed, HandleAgWindowedEinsumLoop(comp, stream_id));
+      TF_ASSIGN_OR_RETURN(changed, HandleAgWindowedEinsumLoop(comp));
       all_ag_loops_.push_back(
           WindowedEinsumAgLoops(comp->WhileCallInstruction()));
+      all_windowed_einsum_loops.push_back(comp->WhileCallInstruction());
     }
```

Comment:
Given my previous suggestion to inline `HandleRsWindowedEinsumLoop` and `HandleAgWindowedEinsumLoop`, I think it makes sense to merge the blocks `if (comp->name().find(kWindowedEinsumRsLoopName) == 0)` and `else if (comp->name().find(kWindowedEinsumAgLoopName) == 0)`